src/app/main-api-handler.ts

```diff
@@ -413,3 +411,16 @@ ipcMain.handle(
     return;
   },
 );
+
+/**
+ * Log API call parameters.
+ */
+const logApiCallParams = (arg: any) => {
+  const apiCmd = arg.cmd;
+  const apiCmdParams = cloneDeep(arg);
```

Comment:
This is will very expensive right? I think we can just select what info we want to log?